spec/controllers/admin/members_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+describe Admin::MembersController do
+  context "when user is authenticated" do
+    before do
+      @request.env["HTTP_AUTHORIZATION"] = "Basic " + Base64::encode64("secretary:password")
```

Comment:
This approach will be susceptible to a brute-force attack, right? There's no rate limiting?

Reply:
> This approach will be susceptible to a brute-force attack, right? There's no rate limiting?

Well, to tell you the truth I have not spend much time trying to come up with a better solution, but once more this PR is just to add the automated tests.

The next one is to update Ruby and Rails.

Later on we can think of something.

But yes, I agree this is not the best solution. Thanks for the review @Aupajo I will work on this one after deploying both PRs on prod :tada::tada::tada:

Reply:
Yeah, I agree: this is an improvement beyond the scope of the app.

Frankly, I think we shouldn't maintain this app. If we're going to replace it, it's hard to justify effort fixing its flaws.

Reply:
> Frankly, I think we shouldn't maintain this app. If we're going to replace it, it's hard to justify effort fixing its flaws.

As the original author of this app and secretary at the time, I'd like to add that this was only really intended as a short-lived UX to populate an official spreadsheet for us while we sorted our documentation out. This has a very Mr. Meeseeks vibe to it.